docs/kubectl-exec.md

```diff
@@ -0,0 +1,60 @@
+## kubectl exec
+
+Execute a command in a container.
+
+### Synopsis
+
+Execute a command in a container.
+Examples:
+  $ kubectl exec -p 123456-7890 -c ruby-container date
+  <returns output from running 'date' in ruby-container from pod 123456-7890>
+
+  $ kubectl exec -p 123456-7890 -c ruby-container -i -t -- bash -il
+  <switches to raw terminal mode, sends stdin to 'bash' in ruby-container from
+   pod 123456-780 and sends stdout/stderr from 'bash' back to the client
+
+kubectl exec -p <pod> -c <container> -- <command> [<args...>]
+
+### Options
+
+```
+  -c, --container="": Target container
+  -p, --pod="": Target pod
```

Comment:
Can you say "name" for this, then (and for container).  Maybe toss a TODO somewhere to accept UIDs, too.